### DISCUSSION TITLE
How to authenticate to Drupal site to use fastBuilds

### DISCUSSION THREAD

**Discussion by Vacilando** [gatsbyjs/gatsby#32025] [Help]
According to the README at https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-drupal/README.md :

"This will require authentication to your Drupal site and a Drupal user with the Drupal permission to sync gatsby fastbuild log entities."

Where/how in Gatsby config can we indicate the relevant Drupal user to use?

**Comment by smthomas** [comment]
You can use the username and password for the Drupal user you want to authenticate as in the HTTP Basic auth config options - https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-drupal/README.md#basic-auth

**Reply by Vacilando** [reply]
Usually, Basic Auth credentials are separate from Drupal login credentials.

Or is there a functionality that passes the Basic Auth credentials to Drupal to log in? Where can we find the documentation?
